src/main/java/net/minecraftforge/client/event/RenderLivingEvent.java

```diff
@@ -21,45 +21,58 @@
 
 import net.minecraftforge.fml.common.eventhandler.Cancelable;
 import net.minecraftforge.fml.common.eventhandler.Event;
+import net.minecraft.client.Minecraft;
 import net.minecraft.client.renderer.entity.RenderLivingBase;
 import net.minecraft.entity.EntityLivingBase;
 
 public abstract class RenderLivingEvent<T extends EntityLivingBase> extends Event
 {
     private final EntityLivingBase entity;
     private final RenderLivingBase<T> renderer;
+    private final float partialRenderTick;
     private final double x;
     private final double y;
     private final double z;
 
+    /**
+     * Use the constructor that includes partialRenderTick instead.
+     */
+    @Deprecated
     public RenderLivingEvent(EntityLivingBase entity, RenderLivingBase<T> renderer, double x, double y, double z)
     {
+        this(entity, renderer, Minecraft.getMinecraft().getRenderPartialTicks(), x, y, z);
+    }
+
+    public RenderLivingEvent(EntityLivingBase entity, RenderLivingBase<T> renderer, float partialRenderTick, double x, double y, double z)
+    {
         this.entity = entity;
         this.renderer = renderer;
+        this.partialRenderTick = partialRenderTick;
         this.x = x;
         this.y = y;
         this.z = z;
     }
 
     public EntityLivingBase getEntity() { return entity; }
     public RenderLivingBase<T> getRenderer() { return renderer; }
+    public float getPartialRenderTick() { return partialRenderTick; }
     public double getX() { return x; }
     public double getY() { return y; }
     public double getZ() { return z; }
 
     @Cancelable
     public static class Pre<T extends EntityLivingBase> extends RenderLivingEvent<T>
     {
-        public Pre(EntityLivingBase entity, RenderLivingBase<T> renderer, double x, double y, double z){ super(entity, renderer, x, y, z); }
```

Comment:
To maintain binary compatibility, this method should be deprecated and not removed.